dockerfiles/Dockerfile.python

```diff
@@ -15,7 +15,7 @@ WORKDIR /app/
 RUN pip install pipenv
 
 # Copy Pipfiles in the container
-COPY Pipfile Pipfile.lock ./
+COPY Pipfile.*  ./
```

Comment:
this fixes the problem that if you delete your local `Pipfile.lock`, and then try to run `docker-catchup` and `docker-new-env`, those commands will error out.

Reply:
nice workaround

Reply:
```suggestion
COPY Pipfile*  ./
```